index.d.ts

```diff
@@ -0,0 +1,47 @@
+import { MutableRefObject, ReactElement, Ref } from "react";
+
+import * as CSS from "csstype";
+
+export as namespace Showtime;
```

Comment:
I don't know if this matters from a TS perspective, but should `Showtime` here (and elsewhere in this file) be changed to `ReactShowtime`, since that's the library's full name? In the context of the library `Showtime` generally refers to the `<Showtime>` component rather than the library.

Reply:
@pcaisse I defer to you on this one, I believe this was a change you added.

Reply:
Did I? I thought you added it :joy: I'm not sure if this is needed since all the types will live in this one file.

Reply:
Oh jeez. Also not confident its necessary. Let's remove for now?

Reply:
Yeah, if the build passes without it let's ditch it.

Reply:
Ah, you edited your comment! :detective: I don't really understand the use case for namespaces unless you're using defining types across files, but then again I don't know that much about namespaces.

Here's another JS-first React project that we use that had TS types added after the fact and it doesn't use a namespace FWIW:
https://github.com/jbetancur/react-data-table-component/blob/master/index.d.ts

I'm fine with renaming it or deleting it.

Reply:
Yeah sorry for the ninja edit. That's always a bad idea. Good enough of an example for me. Removing.